server/util/bazelisk/BUILD

```diff
@@ -0,0 +1,24 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+
+genrule(
+    name = "bazelisk-1.10.1_crossplatform",
+    srcs = select({
+        "@bazel_tools//src/conditions:darwin_x86_64": ["@io_bazel_bazelisk-1.10.1-darwin-amd64//file:downloaded"],
+        "@bazel_tools//src/conditions:darwin_arm64": ["@io_bazel_bazelisk-1.10.1-darwin-arm64//file:downloaded"],
+        "//conditions:default": ["@io_bazel_bazelisk-1.10.1-linux-amd64//file:downloaded"],
```

Comment:
Does this end up downloading only the active condition, or all of them (https://github.com/bazelbuild/bazel/issues/12861)? With toolchains it would for sure only download the active condition. https://github.com/keith/buildifier-prebuilt is an example for buildifier.

Reply:
I tried running a clean build on a Linux machine, and it looks like we fetch only the linux-amd64 build: https://app.buildbuddy.io/invocation/191db983-3c49-4880-8e43-6296edc0241e#fetches